48 modifications based on aw feedback (#50)
siamakn/temp-nomad-onboarding-workshop: docs/Module2/M2_x_accessing_NOMAD.md

# Navigating to NOMAD

> The [NOMAD landing page](https://nomad-lab.eu/nomad-lab/){:target="_blank"} provides an overview of different NOMAD solutions as well as its features and functionalities, along with links to documentation, tutorials, and project history. This is the starting point for accessing the NOMAD, where you can upload, manage, and explore data.

Reviewer: I suggest to remove the lines describing the NOMAD webpage. It is part of promotion, rather than on-boarding. The documentation and tutorials have been already mentioned in "getting support section".

> The [NOMAD landing page](https://nomad-lab.eu/nomad-lab/){:target="_blank"} provides an overview of different NOMAD solutions as well as its features and functionalities, along with links to documentation, tutorials, and project history. This is the starting point for accessing the NOMAD, where you can upload, manage, and explore data.

Reviewer: ```suggestion
The [NOMAD landing page](https://nomad-lab.eu/nomad-lab/){:target="_blank"} is the starting point for accessing NOMAD, where you can upload, manage, and explore data.
```

Reviewer: Lets replace the NOMAD landing page with the NOMAD website. 
The reason I'm suggesting this is that we start to describe the NOMAD landing page on the GUI section as the NOMAD software landing page. This can be confusing.

![NOMAD Landing Page](images/NOMAD_landing_page.png)

> You can access the official NOMAD by clicking on the blue **OPEN NOMAD**

Reviewer: ```suggestion
To access the latest stable version of NOMAD (official release), click on the blue **OPEN NOMAD** button on the top right of the page.
```

> button on the top right of the page.

Reviewer: ```suggestion

```

> However, there are other access points to NOMAD. While in this Module we mainly focus on using the official NOMAD release, it is worth to have a brief overview of other NOMAD installations. In order to see the list of these installations, scroll down the The [NOMAD landing page](https://nomad-lab.eu/nomad-lab/){:target="_blank"} to reach the bottom of the page, and locate **Installations** header at the right side of the bottom blue ribbon.

Reviewer: I highly recommend to hide this out, in a box for example. I don't see the beginner user, who is attending the on-boarding workshop, as being interested in knowing about all these versions. This can be confusing and too much information.

My suggestion is to combine the pages M2_x_accessing_NOMAD and M2_NOMAD_GUI.
